lib/class-wp-theme-json.php

```diff
@@ -166,6 +166,7 @@ class WP_Theme_JSON {
 				'customTextTransforms'  => null,
 			),
 			'custom'     => null,
+			'layout'     => null,
```

Comment:
you'll notice that this property is defined in the "theme.json" of the empty theme I shared. It's just a temporary position and naming... The idea is just to show that we can use theme.json to define a layout that group block can use using a single click.